setup.cfg

```diff
@@ -9,12 +9,11 @@ license = BSD+Patent
 license_file = LICENSE.md
 url = https://www.plasmapy.org
 project_urls =
-    Documentation = https://docs.plasmapy.org/
-    Changes = https://docs.plasmapy.org/en/stable/whatsnew/index.html
-    Source Code = https://github.com/plasmapy/plasmapy
-    Issue Tracker = https://github.com/plasmapy/plasmapy/issues/
-    Twitter = https://twitter.com/PlasmaPy
-    Chat = "https://app.element.io/#/room/#plasmapy:openastronomy.org"
+    Documentation=https://docs.plasmapy.org/
+    Changelog=https://docs.plasmapy.org/en/stable/whatsnew/index.html
+    Source=https://github.com/plasmapy/plasmapy
+    Issues=https://github.com/plasmapy/plasmapy/issues/
+    Twitter=https://twitter.com/PlasmaPy
```

Comment:
I had to modify these, notably removing the Chat one, as Twine was giving me problems on upload - as documented at https://github.com/pypa/warehouse/issues/10198.